### PR TITLE
Python: Fix multi-instance parsing

### PR DESCRIPTION
**Pull request by BrainFPV**
Fixes Python log parsing for logs that have multi-instance UAVOs present.

### CODE REVIEW

File: python/taulabs/telemetry.py [hunk 1]
```diff
@@ -54,7 +54,9 @@ def __init__(self, githash=None, service_in_iter=True,
         if githash:
             uavo_defs.from_git_hash(githash)
         else:
-            uavo_defs.from_uavo_xml_path("shared/uavobjectdefinition")
+            xml_path = os.path.join(os.path.dirname(__file__), "..", "..",
+                                    "shared", "uavobjectdefinition")
+            uavo_defs.from_uavo_xml_path(xml_path)
```
Comment (BrainFPV) [inline python/taulabs/telemetry.py]: Added this so the `taulabs` module can be imported from any directory.

**Comment by mlyle** [comment]
LGTM-- sorry I didn't get to this earlier.

As long as you're doing this, though, you should revert 722ac0beef1d59d333c0289ea1067a5f3b8087f in the same PR, as that's a workaround for reading instanceid as a timestamp.

**Comment by mlyle** [comment]
Also consider renaming the 'cleanup' log message.

**Comment by BrainFPV** [comment]
@mlyle thanks for reviewing. Should be good now :)

**Comment by mlyle** [comment]
Tests good to me.. i'll factor this down into the objects later to pick up some speed.

**Pull request merged**